### PR TITLE
Fixes #12274: Rudder-tests offline

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/12274

### CODE REVIEW

File: platforms/FDA.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (amousset) [inline platforms/FDA.json]: We may not need this one in the repo :laughing:

**Comment by Fdall** [comment]
Commit modified

File: rtf [hunk 2]
```diff
@@ -185,14 +185,17 @@ def have_proxy():
...
+  try:
+    nrm_ip = socket.gethostbyname("republique-1.normation.com.")
+    req = requests.get("http://ipinfo.io/ip")
+    if req.status_code == 200:
+      if req.content.strip() == nrm_ip:
+        use_proxy = proxy_line
+        return use_proxy
+  except socket.error as e:
+    print("Network unreachable, can only start local already set up vms")
+    return use_proxy
```
Comment (peckpeck) [inline rtf]: You removed use_proxy = "" and indented the return

File: rtf [hunk 3]
```diff
@@ -298,7 +301,18 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
-    os.system(proxy + "vagrant up " + self.hostid + " --provider="+self.provider)
+    if (proxy==None):
```
Comment (peckpeck) [inline rtf]: I'd say this is over engineered, the empty proxy variable is sufficient for all the cases

**Comment by Fdall** [comment]
Commit modified

File: rtf [hunk 4]
```diff
@@ -185,16 +185,22 @@ def have_proxy():
...
+    nrm_ip = socket.gethostbyname("republique-1.normation.com.")
+    req = requests.get("http://ipinfo.io/ip")
+    if req.status_code == 200:
+      if req.content.strip() == nrm_ip:
+        use_proxy = proxy_line
+        return use_proxy
+  except socket.error as e:
+    print("Network unreachable, can only start local already set up vms")
+    use_proxy=""
+    return use_proxy
```
Comment (peckpeck) [inline rtf]: duplicate of the lines below

File: rtf [hunk 5]
```diff
@@ -298,6 +304,15 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
+    if (proxy==""):
+      command = "vagrant status " + self.hostid
```
Comment (peckpeck) [inline rtf]: i don't understand what this is for

Comment (Fdall) [reply rtf]: When offline we can only try to start a group of machines that are shutdown.
This ensure the requested vms are just in poweroff mode and not in the need to be installed completely.

**Comment by peckpeck** [comment]
there are conflicts, you need to run rudder-dev rebase

**Pull request closed without merging**